### PR TITLE
sorted the desserts in alphabetical order

### PR DESCRIPTION
**Pull request by uwaseem**
Sorted the dessert in alphabetical order. As you can see, cake was double prior to this. Now we have a clean list of 20 desserts. Might be worth moving this list into its own separate file.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const utils = require('./helpers/utils');
 
-const desserts = ['akok', 'apple crumble', 'cake', 'cempedak creme', 'cempedak fritters', 'cekodok','cendol','ais kacang','eclair','chocolate','cake','tart', 'muffin', 'brownies', 'cupcake', 'frgál', 'kremrole', 'větrník', 'buchty', 'krapfen', 'kek lapis']
+const desserts = ['ais kacang', 'akok', 'apple crumble', 'brownies', 'buchty', 'cake', 'cekodok', 'cempedak creme', 'cempedak fritters', 'cendol', 'chocolate', 'cupcake', 'eclair', 'frgál', 'kek lapis', 'krapfen', 'kremrole', 'muffin', 'tart', 'větrník']
```
Comment (sanijalal) [inline index.js:3]: What's a buchty?

**Review by sanijalal — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (uwaseem) [reply index.js:3]: existing entry mate, I have no idea.

Comment (uwaseem) [reply index.js:3]: Ah yes, these are actually pretty delicious.
https://en.wikipedia.org/wiki/Buchteln

Comment (uwaseem) [reply index.js:3]: This was added by you actually, last year.
https://github.com/sanijalal/random-dessert-generator/pull/3/files

Comment (sanijalal) [reply index.js:3]: lol, I don't remember. Thanks for the refresher.

**Pull request merged**